dll/win32/shell32/precomp.h

```diff
@@ -45,6 +45,10 @@
 #undef ShellExecute
 #include <undocshell.h>
 
+// #if (_WIN32_WINNT >= 0x0600)
```

Comment:
This is commented for the moment, because:
- our shell32 is built with _WIN32_WINNT set to 0x600 (Vista+),
- but our exports (especially regarding ShellMessageBoxA/W) are Win2003-compatible (they change starting Vista+, as explained in the commit log and the code).

Reply:
So what is the #ifdef doing here?
Even in 2k3 mode shell should use the Wrap function internally?

Reply:
So yes, in 2k3 mode it implicitly uses the Wrap function internally. But in Vista+, where shlwapi exports named versions of ShellMessageBoxA and W, the shell32 ones directly redirects to these shlwapi exports, and it doesn't use the Wrap version internally anymore.

Reply:
The #if should actually read `#if (_WIN32_WINNT < 0x0600)`